### ISSUE TITLE
failover: enable different dns discovery domain for AD subdomains

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2397]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2393
- Created at 2014-07-30 11:55:21 by [pbrezina](https://pagure.io/user/pbrezina)
- Closed at 2020-03-24 14:10:33 as wontfix
- Assigned to nobody
---

Current SRV AD plugin does not allow to set different dns discovery domains for trusted subdomains if dns_discovery_domain is set in sssd.conf. Therefore if the option is present, we always look up directory controllers in the domain configured in dns_discovery_domain. This means that the obtained servers for subdomains are wrong and we are not able to look up users and groups (unless the found dc is also gc).

### Comments

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-07-30 12:23:23](https://pagure.io/SSSD/sssd/issue/2393#238074)**

Fields changed

blocking:  => #2394

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-08-06 09:41:27](https://pagure.io/SSSD/sssd/issue/2393#238075)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.13 beta

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-08-06 09:43:50](https://pagure.io/SSSD/sssd/issue/2393#238076)**

Fields changed

rhbz:  => 0

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-09-26 18:50:00](https://pagure.io/SSSD/sssd/issue/2393#238077)**

Fields changed

mark:  => 0

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2015-02-10 14:37:48](https://pagure.io/SSSD/sssd/issue/2393#238078)**

This would be nice to have, but requires quite a bit of refactoring of the failover code.

milestone: SSSD 1.13 beta => SSSD 1.13 backlog

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2015-02-12 20:13:38](https://pagure.io/SSSD/sssd/issue/2393#238079)**

Mass-moving tickets not planned for the 1.13 release to 1.14

milestone: SSSD 1.13 backlog => SSSD 1.14 beta

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2016-02-16 12:12:44](https://pagure.io/SSSD/sssd/issue/2393#238080)**

This would be nice to have, but the changes required for the failover and responder refactoring are not scoped for the 1.14 release.

milestone: SSSD 1.14 beta => SSSD 1.15 beta
sensitive:  => 0

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2017-02-24 14:57:31](https://pagure.io/SSSD/sssd/issue/2393#238081)**

**Metadata Update from @pbrezina**:
- Issue marked as blocked by: #2394
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-13 12:27:18](https://pagure.io/SSSD/sssd/issue/2393#633125)**

**Metadata Update from @thalman**:
- Custom field blocking reset (from #2394)
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: Canditate to close

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:10:32](https://pagure.io/SSSD/sssd/issue/2393#636588)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:10:35](https://pagure.io/SSSD/sssd/issue/2393#636589)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
